tasks/drupal.yml

```diff
@@ -71,6 +71,8 @@ tasks:
           ./vendor/bin/drush {{.site}} config:status
           exit 1;
         fi
+      # Run cron after updating to refresh items.
+      - ./vendor/bin/drush {{ .site }} cron
```

Comment:
I'm in favor of adding `--verbose` to this. I've found when it fails (which is one of the reasons why we're running this here) we don't get a lot of details about what specifically failed without having `--verbose`